crates/router/src/connector/authorizedotnet/transformers.rs

```diff
@@ -1359,7 +1373,10 @@ impl TryFrom<&AuthorizedotnetRouterData<&types::PaymentsCompleteAuthorizeRouterD
                 .payer_id;
         let transaction_type = match item.router_data.request.capture_method {
             Some(enums::CaptureMethod::Manual) => TransactionType::ContinueAuthorization,
-            _ => TransactionType::ContinueCapture,
+            Some(enums::CaptureMethod::Automatic)
+            | Some(enums::CaptureMethod::ManualMultiple)
+            | Some(enums::CaptureMethod::Scheduled)
```

Comment:
We should throw 4xx in case of ManualMultiple & Scheduled

Reply:
Done